### PR TITLE
[Android] Fix: Change navigation bar to theme color

### PR DESCRIPTION
**Pull request by walmat**
Just a simple QOL update to Android. Enables fullscreen mode by default and swiping up from the bottom enables the Navigation bar. Swiping down from the top enables the Statusbar.

https://github.com/rainbow-me/rainbow/assets/6284525/6ced37d5-280c-4f5c-9f7b-ce85c1757efb

### CODE REVIEW

**Comment by walmat** [comment]
Just a simple QOL update to Android. Enables fullscreen mode by default and swiping up from the bottom enables the Navigation bar. Swiping down from the top enables the Statusbar.



https://github.com/rainbow-me/rainbow/assets/6284525/6ced37d5-280c-4f5c-9f7b-ce85c1757efb

**Comment by walmat** [comment]
@dereknelson not sure if this will collide with Expo stuff at all. I know expo has a simpler way of changing the default behavior. We could maybe hold off on this until expo is done if that's preferred.

**Comment by dereknelson** [comment]
> @dereknelson not sure if this will collide with Expo stuff at all. I know expo has a simpler way of changing the default behavior. We could maybe hold off on this until expo is done if that's preferred.

i don't think it would collide, expo has their [status bar](https://docs.expo.dev/versions/latest/sdk/status-bar/) package but what you have here is essentially that, i say it's worth shipping this

**Review by jinchung — APPROVED** [review]
🌮  code review only

**Review by ibrahimtaveras00 — COMMENTED** [review]
Looks like a bit of space at the bottom of these two sheets 
image

image

**Review by ibrahimtaveras00 — APPROVED** [review]
👍🏽

File: src/components/sheet/SlackSheet.js [hunk 1]
```diff
@@ -123,7 +125,7 @@ export default forwardRef(function SlackSheet(
   const { colors } = useTheme();
   const contentContainerStyle = useMemo(
     () => ({
-      paddingBottom: bottomInset,
+      paddingBottom: bottomInset + 128,
```
Comment (walmat) [inline src/components/sheet/SlackSheet.js:128]:
```suggestion
      paddingBottom: bottomInset,
```

**Pull request merged**